python/lsst/cp/pipe/astierCovPtcUtils.py

```diff
@@ -424,16 +425,15 @@ def getFitDataFromCovariances(i, j, mu, fullCov, fullCovModel, fullCovSqrtWeight
     covarianceModel = fullCovModel[:, i, j]*(gain**2)
     weights = fullCovSqrtWeights[:, i, j]/(gain**2)
 
-    # select data used for the fit
-    mask = weights != 0
+    maskFromWeights = weights != 0
     if returnMasked:
```

Comment:
Is this case ever called now?  Am I mistaken that the turnoff based masking is the only masking used?

Reply:
It's not called now.  Now we do use the final mask that results after doing the {EXPAPPROXIMATION} fit before this {FULLCOVARIANCE} fit.  This final mask has cuts from outlier rejection (in getInitialGoodPoints) and iterations ("newMask" in the code), in addition to the turn-off based masking (which is included in the detInitialGoodPoints function). 
I decided to leave this original weighting scheme from the Astier code in case that we want to use it (so I change the name of the variable slightly), but we may decide in the future that we want to get rid of this if we don't use it at all to make things easier to read.